packages/dappmanager/test/integration/migrations/composeMigration.test.int.ts

```diff
@@ -2,103 +2,127 @@ import { expect } from "chai";
+          - ${serviceName}.test-migration.dappnode
+          - ${serviceName}.dappnode`;
 
   const composeToBeMigratedBefore = `
 version: '3.4'
 networks:
-  dncore_network:
-    name: dncore_network
+  ${dncoreNetwork}:
+    name: ${dncoreNetwork}
     external: true
 services:
-  dappmanager.dnp.dappnode.eth:
-    image: "chentex/random-logger"
-    container_name: DAppNodeCore-dappmanager.dnp.dappnode.eth
+  ${serviceName}:
+    image: ${randomImage}
+    container_name: ${containerName}
     restart: always
     dns: 172.33.1.2
     networks:
-      dncore_network:
+      ${dncoreNetwork}:
         ipv4_address: 172.33.1.7`;
 
-  const dncoreNetwork = params.DNP_PRIVATE_NETWORK_NAME;
-  const containerName = "DAppNodeCore-dappmanager.dnp.dappnode.eth";
-  const randomImage = "chentex/random-logger";
-  const testMigrationPath = process.cwd() + "/test/integration/migrations";
-
   before("Run random container", async () => {
     // Create compose
-    await shellSafe(`mkdir ${testMigrationPath}/test-migration`);
+    await shellSafe(`mkdir -p ${testPackagePath}`);
     // Compose to be migrated
     fs.writeFileSync(
-      `${testMigrationPath}/test-migration/docker-compose.yml`,
+      `${testPackagePath}/docker-compose.yml`,
       composeToBeMigratedBefore
     );
-    // Compose already migrated
-    fs.writeFileSync(
-      `${testMigrationPath}/test-migration/docker-compose-migrated.yml`,
-      composeAlreadyMigrated
-    );
-    // Redeclare global variables
-    params.DNCORE_DIR = testMigrationPath;
-    params.REPO_DIR = testMigrationPath;
+
     // Startup container
     await shellSafe(
-      `docker-compose -f ${testMigrationPath}/test-migration/docker-compose.yml -p DNCORE up -d`
+      `docker-compose -f ${testPackagePath}/docker-compose.yml -p DNCORE up -d`
     );
-    const containerkExists = await shellSafe(
+    const containerExists = await shellSafe(
       `docker container ls --filter name=${containerName}`
     );
 
     const networkExists = await shellSafe(
       `docker network ls --filter name=${dncoreNetwork}`
     );
 
-    if (!containerkExists || !networkExists)
+    if (!containerExists || !networkExists)
       throw Error("Error creating container or/and dncore_network");
   });
 
   it("Should do alias migration in compose", async () => {
-    const aliases = ["dappmanager.dnp.dappnode.eth.test-migration.dappnode", "dappmanager.dappnode"];
+    const aliases = ["test.test-migration.dappnode", "test.dappnode"];
     migrateCoreNetworkAndAliasInCompose(container, aliases);
 
     const composeAfter = fs.readFileSync(
-      `${testMigrationPath}/test-migration/docker-compose.yml`,
+      `${testPackagePath}/docker-compose.yml`,
       { encoding: "utf8" }
     );
     expect(composeAfter.trim()).to.equal(composeAlreadyMigrated.trim());
-  });    
+  });
+
+  it("Should remove DNS from compose file", async () => {
+    removeDnsFromPackageComposeFile(container.dnpName, false);
```

Comment:
should we test also the `removeDnsFromPackageComposeFile` for core packages or is not worth it?